internal/test/examples.go

```diff
@@ -60,6 +60,7 @@ func ExampleEtcdPeer(namespace string) *v1alpha1.EtcdPeer {
 						},
 					},
 				},
+				InitialClusterState: v1alpha1.InitialClusterStateNew,
```

Comment:
Yeah, I think we should add validation to ensure that this field is always set.

Reply:
I think it has to be. This test failed without this.